tool/services/github_service.py

```diff
@@ -163,6 +163,18 @@ def update_labels(self, labels):
                 changes += 1
         return changes
 
+    def list_tags(self):
+        """
+        Get list of tags in the repository.
+        :return:  [Tags]
+        """
+        tags = []
+        for tag in self.repo.get_tags():
+            tags.append({'name': tag.name,
+                         'url': f"https://github.com/{self.repo.full_name}"
+                                f"/release/tag/{tag.name}"})
```

Comment:
```suggestion
                                f"/releases/tag/{tag.name}"})
```

Otherewise LGTM, thanks!

Reply:
Fixed. But for some reason release works too.